src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/ClientBaseTests.cs

```diff
@@ -125,31 +125,92 @@ public static void ClientMessageInspector_Verify_Invoke()
 
```

Comment:
Incidentally, what I forgot to say last review is that this is great to see this level of unit tests for these types.  Nice coverage.